packages/block-library/src/search/edit.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { RichText, InspectorControls } from '@wordpress/editor';
+import { PanelBody, TextControl } from '@wordpress/components';
+
+export default function SearchEdit( { attributes, setAttributes } ) {
+	const { label, placeholder } = attributes;
+
+	return (
+		<Fragment>
+			<div className="wp-block-search">
```

Comment:
Should it be also using `form` to ensure that CSS is properly applied if someone uses a tag name specific selectors?

Reply:
Is that likely? We already have the problem of the `<label>` being a `<div>` because of some `RichText`. Making this a `<form>` means that we'd have to add a `onSubmit={ ( event ) => event.stopPropagation() }` which I find a little distracting.

Reply:
I don't know what is the best way moving forward, I just wanted to raise this so we could discuss and come up with something solid as this is going to be replicated in other places :)